src/WinGetUtil/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="cpprestsdk.v142" version="2.10.15" targetFramework="native" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.200729.8" targetFramework="native" />
   <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.200519.2" targetFramework="native" />
 </packages>
```

Comment:
The same thing applies to a number (but not all) of the files here.

Tangentially related, the spelling bot is similarly upset about the patterns file below (I can't comment on that file because it isn't part of the PR), although for some reason (I need to investigate) it can't seem to properly count to the last line of the file...